flink-dist/pom.xml

```diff
@@ -204,6 +204,11 @@ under the License.
 				</exclusion>
 			</exclusions>
 		</dependency>
+		
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-shaded-netty</artifactId>
+		</dependency>
```

Comment:
This may be a relic of a previous version where all other dependency declarations were set to `<provided>`.